Src/Installer/Dasher.wxs

```diff
@@ -18,11 +18,13 @@
 					<Directory Id="INSTALLDIR" Name="Dasher 5.00">
 						<Component Id="MainExecutable" Guid="5a93ba53-2e2d-4f88-8a1f-5bdd8f130394">
 							<File Id="DasherEXE" Name="Dasher.exe" DiskId="1" Source="..\Win32\Target\Release\Dasher.exe" Vital="yes"/>
-							<File Id="ExpatDLL" Name="libexpat.dll" DiskId="1" Source="$(env.EXPAT)\bin\libexpat.dll" Vital="yes"/>
-						</Component>
-						<Component Id="Help" Guid="cf486329-8d04-462b-afd4-e8e0bccc9c95">
-							<File Id="HelpFile" Name="Dasher.chm" DiskId="1" Source="..\..\Data\Help\Win32\dasher.chm"/>
+							<File Id="ExpatDLL" Name="libexpat.dll" DiskId="1" Source="$(env.EXPAT)\Bin\libexpat.dll" Vital="yes"/>
```

Comment:
Do we need the change in this line (bin -> Bin)?

Reply:
I will go with answering yes. Because there is no folder "bin" in Expat folder. There's a "Bin" and it contains libexpat.dll. However it couldn't solve the copy issue for .dll. So, doesn't exactly matter now, but true one is "Bin".